src/theory/quantifiers/ceg_instantiator.h

```diff
@@ -40,52 +40,49 @@ class CegqiOutput {
 
 class Instantiator;
 
-
-//TODO: refactor pv_coeff to pv_prop throughout
-//generic class that stores properties for a variable to solve for in CEGQI
+//stores properties for a variable to solve for in CEGQI
+//  For LIA, this includes the coefficient of the variable, and the bound type for the variable
 class TermProperties {
 public:
-  TermProperties() : d_type(-1) {}
+  TermProperties() : d_type(0) {}
+  // type of property for a term
+  //  for arithmetic this corresponds to bound type (0:equal, 1:upper bound, -1:lower bound)
```

Comment:
Yes I agree.  In further refactoring, I will make TermProperties a virtual class so that the infrastructure is truly independent of LIA.  This requires some thought but if I do that I will definitely turn bound type into an enum in the specialized "TermPropertiesArith" class.   I'll do this in a separate commit.